### ISSUE TITLE
Adding new code

### ISSUE THREAD

**Issue by ick003** [openvolley/ovscout#13]
- [x] Offer the option to add the code before or after the current code.
- [ ] Or, If entering code just prior to a rotation (*z1-*z6 or az1-az6, etc) ensure the copied row is the one above, not the selected one.

**Comment by raymondben** [comment]
'Insert' key to insert new code above current row (as is now), and 'Shift-Insert' to insert new code below?

We also need a 'point validator' that will add/remove auto codes as required in a point (e.g. a point was originally scouted without a losing action, so there will be a generic `*$$&H=` ("losing action") auto code. But then if I edit it to add e.g. a dig error, that auto code should be removed.)

**Comment by ick003** [comment]
Yeah, that would work. Should we add buttons in addition to the key (for all action I mean, insert, delete, edit, add sub, etc)?

**Comment by raymondben** [comment]
I think that's a broader discussion around alternate interfaces (do we do a graphical pointy-clicky style one as a completely separate interface to the typing one?)

**Comment by raymondben** [comment]
Commit ab70a35c821755726788aa9115597ba2da270418 allows [Shift-Insert] to insert code below (uses current line as template), or just [Insert] to insert new code above (uses previous line as template)
